### DISCUSSION TITLE
Como testar no Chrome Android APIs que só funcionam em ambiente seguro HTTPS?

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#284] [Perguntas e Respostas]
Eu gostumo fazer um servidor com o módulo `http-server`, e acesso meu IP no Android, o problema é que o Chrome parou de suportar algumas APIs como a Geolocation [geolocation on secure contexts only](https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only), pois reclama de não estar em ambiente seguro.

Embora no browser o localhost ele permita usar a API, usando o IP ele não reconhece como localhost, não sei se foi eu que li pouco, ou se o próprio link acima em algum link sugere uma solução.

Como que vocês estão fazendo para testar no Chrome Android funções como Camera, Audio e Video, Geolocation entre outras em um IP local?

Um amigo me recomendou um serviço como o https://ngrok.com/ mas queria saber alternativas, tentei criar um certificado local e usa-lo no http-server, mas não sou muito bom de lidar com certificados e não consegui...

Enfim, poderiam me ajudar? Abraço!

**Comment by icaromh** [comment]
Ia te sugerir criar um proxy reverso a partir de um servidor externo, apontando pro teu ip local. Mas tu teria que configurar certificados da mesma forma :/

**Comment by felquis** [comment]
Pois é, e o ngrok para usar tls precisa pagar :/ 

Continuo precisando de uma solução agil
